src/AspNet.Security.OpenId/OpenIdAuthenticationPostConfigureOptions.cs

```diff
@@ -0,0 +1,114 @@
+using System;
+using System.Net.Http;
+using System.Text;
+using AngleSharp.Parser.Html;
+using AspNet.Security.OpenId;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.DataProtection;
+using Microsoft.Extensions.Options;
+using Microsoft.IdentityModel.Protocols;
+
+namespace Microsoft.Extensions.DependencyInjection
+{
+    public class OpenIdAuthenticationPostConfigureOptions<TOptions, THandler> : IPostConfigureOptions<TOptions>
```

Comment:
See my earlier remarks about the name of this class.

Reply:
already renamed to `OpenIdAuthenticationPostConfigureOptions`

Reply:
I'll rename it to `OpenIdAuthenticationInitializer` myself.